### ISSUE TITLE
Create garden project 

### ISSUE THREAD

**Issue by afritzler** [gardener/garden-setup#39]
**What would you like to be added**:
In the initial setup we should create a garden project and also create the corresponding labels on the garden namespace.

garden namespace
```
labels:
  garden.sapcloud.io/role: project
  project.garden.sapcloud.io/name: garden
```

**Why is this needed**:
To create seeds and soils in the garden namespace.

**Comment by Diaphteiros** [comment]
Unfortunately, this is somehow blocked by https://github.com/gardener/gardener/issues/879. The `garden` project needs to have a namespace that is also called `garden` for the `shoot-as-seed` annotations to work - that `garden` namespace does already exist and it contains parts of the Gardener deployment. If we link a project to that namespace, we cannot undo that linking without deleting the namespace and thus destroying the Gardener itself.

**Comment by Diaphteiros** [comment]
Resolved with #265 
(I somehow forgot to close the issue)
